backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5752750_sys_BusinessRule_PP_Product_BOM_Verified.sql

```diff
@@ -0,0 +1,80 @@
+;
+
+-- Value: PP_Product_BOM_NotVerified
+-- 2025-04-24T14:44:09.119Z
+UPDATE AD_Message SET EntityType='D',Updated=TO_TIMESTAMP('2025-04-24 14:44:09.119000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Message_ID=545538
+;
+
+-- Value: PP_Product_BOM_NotVerified
+-- 2025-04-24T14:44:23.074Z
+UPDATE AD_Message_Trl SET MsgText='Die Stücklisten Version muss noch verifiziert werden.',Updated=TO_TIMESTAMP('2025-04-24 14:44:23.074000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Message_ID=545538
+;
+
+-- 2025-04-24T14:44:23.075Z
+UPDATE AD_Message base SET MsgText=trl.MsgText, Updated=trl.Updated, UpdatedBy=trl.UpdatedBy FROM AD_Message_Trl trl  WHERE trl.AD_Message_ID=base.AD_Message_ID AND trl.AD_Language='de_CH' AND trl.AD_Language=getBaseLanguage()
+;
+
+-- Value: PP_Product_BOM_NotVerified
+-- 2025-04-24T14:44:23.619Z
+UPDATE AD_Message_Trl SET IsTranslated='Y',Updated=TO_TIMESTAMP('2025-04-24 14:44:23.619000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Message_ID=545538
+;
+
+-- Value: PP_Product_BOM_NotVerified
+-- 2025-04-24T14:44:27.997Z
+UPDATE AD_Message_Trl SET IsTranslated='Y',Updated=TO_TIMESTAMP('2025-04-24 14:44:27.997000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Message_ID=545538
+;
+
+-- Value: PP_Product_BOM_NotVerified
+-- 2025-04-24T14:44:29.422Z
+UPDATE AD_Message_Trl SET MsgText='Die Stücklisten Version muss noch verifiziert werden.',Updated=TO_TIMESTAMP('2025-04-24 14:44:29.421000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Message_ID=545538
+;
+
+-- 2025-04-24T14:44:29.428Z
+UPDATE AD_Message base SET MsgText=trl.MsgText, Updated=trl.Updated, UpdatedBy=trl.UpdatedBy FROM AD_Message_Trl trl  WHERE trl.AD_Message_ID=base.AD_Message_ID AND trl.AD_Language='de_DE' AND trl.AD_Language=getBaseLanguage()
+;
+
+-- Name: M_Product_BOM verified
+-- 2025-04-24T14:48:00.117Z
+INSERT INTO AD_Val_Rule (AD_Client_ID,AD_Org_ID,AD_Val_Rule_ID,Created,CreatedBy,EntityType,IsActive,Name,Type,Updated,UpdatedBy) VALUES (0,0,540717,TO_TIMESTAMP('2025-04-24 14:48:00.113000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'U','Y','M_Product_BOM verified','S',TO_TIMESTAMP('2025-04-24 14:48:00.113000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
+
+-- Name: M_Product_BOM verified
+-- 2025-04-24T14:48:04.511Z
+UPDATE AD_Val_Rule SET Code='EXISTS (SELECT 1
+              FROM pp_product_bom bom
```

Comment:
why are u selecting again from `pp_product_bom bom`  when u already have the pp_product_bom record available?!